documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,67 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can install and create script for IntelliJ as described in
+
+`https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
+
+. After starting IntelliJ, You **need** to import the cobigen folder as maven project
+
+. You now **need** to configure maven settings in IntelliJ as follows
+
+.. Go to `File`-> `Settings` -> `Build, Execution, Deployment` -> `Build Tools` -> `Maven`
+
+... **Choose** `(Maven home Path)` from your devonFolder -> devonProjects/software/Maven
+
+... **Override** `(user settings file)` -> devonProjects/conf/.m2/settings.xml
+
+... **Override** `(Local repository)` -> devonProjects/conf/.m2/repository
+
+
+image::images/howtos/intellij/maven-settings[Maven Settings]
+
+.. Now go to `File` -> `Project structure` -> `Project Settings` -> `Project add java sdk version from` -> `devonProjects/software/java`
+
+image::images/howtos/intellij/java-sdk.png[Java SDK]
+
+After you have finished your configuration, run maven install and maven clean within IntelliJ. CobiGen should build without any errors.
```

Comment:
maven install and maven clean should be put into parenthesis.

Reply:
backticks